.circleci/config.yml

```diff
@@ -1571,18 +1567,32 @@ parameters:
     default: "9999"
     type: string
 
+  run_nightly_workflow:
+    default: false
+    type: boolean
+
 # -------------------------
-#        WORK FLOWS
+#        WORKFLOWS
+#
+#  When creating a new workflow, make sure to include condition:
+#
+#  when:
+#    and:
+#      - equal: [ false, << pipeline.parameters.run_release_workflow >> ]
+#      - equal: [ false, << pipeline.parameters.run_nightly_workflow >> ]
```

Comment:
This sounds a bit dangerous. Someone has to read carefully the file to learn this. Maybe not to be addressed in this PR, but can we find a way to do it the other way around? 🤔

Reply:
The setup was suboptimal also before. I think we should potentially move to a single "string" parameter. Still I really dislike CircleCI setup here as it makes understanding the workflow logic really complicated.